tree/src/package/mod.rs

```diff
@@ -0,0 +1,80 @@
+mod error;
+pub use error::{PackageError, PackageErrorType};
+
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Package {
```

Comment:
There are some issues with this `struct`:
- `PKGEPOCH` needs to be represented
- Different architectures need different sets of dependencies, recommend using a `HashMap<String, Vec<String>`
- `FAIL_ARCH` needs to be represented
- When used with an automated build system, `SRCS` and `SUBDIR` needs to be represented